src/components/analyser/search/DashboardCards.tsx

```diff
@@ -66,52 +69,51 @@ export const DashboardCardsRow: React.FC<DashboardCardsRowProps> = ({
   stock,
   selectStock,
 }) => {
-  const [hover, setHover] = React.useState<boolean>(false);
   const classes = useStyles();
   const { t } = useTranslation();
 
   return (
-    <Card
-      className={classes.card}
-      style={{ width: 280, height: 450, margin: 10 }}
-    >
-      <CardMedia
-        style={{
-          height: 200,
-          width: 260,
-          placeItems: 'center',
-          alignItems: 'center',
-          objectFit: 'scale-down',
-          margin: 10,
+    <Card className={classes.card} style={{ width: 280, margin: 10 }}>
+      <ButtonBase
+        className={classes.cardAction}
+        onClick={() => {
+          selectStock(stock.symbol);
         }}
-        component="img"
-        image={stock.picture.toString()}
-      />
-      <CardContent className={classes.content}>
-        <Typography
-          className="MuiTypography--heading"
-          variant="h6"
-          gutterBottom
-        >
-          {stock.symbol}
-        </Typography>
-        <Typography className="MuiTypography--subheading" variant="caption">
-          {stock.name}
-        </Typography>
-        <Divider className={classes.divider} light />
-        <Button
-          size="small"
-          color="primary"
-          onClick={() => {
-            selectStock(stock.symbol);
+      >
+        <CardMedia
+          style={{
```

Comment:
move styles to styles section (const useStyles)

Reply:
done